config-generators/cosmosdb_nosql-commands.txt

```diff
@@ -1,8 +1,14 @@
 init --config "dab-config.CosmosDb_NoSql.json" --database-type "cosmosdb_nosql" --connection-string "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==" --cosmosdb_nosql-database "graphqldb" --cosmosdb_nosql-container "planet" --graphql-schema "schema.gql" --host-mode Development --cors-origin "http://localhost:5000"
 add Planet --config "dab-config.CosmosDb_NoSql.json" --source "graphqldb.planet" --permissions "anonymous:create,read,update,delete" --graphql "Planet:Planets"
+update Planet --config "dab-config.CosmosDb_NoSql.json" --permissions "anonymous:read" --fields.include "*"
```

Comment:
Do you need to explicitly add the `field.include *` property now to be able to read all fields?

If the fields property is not specified, will it assume read permission for all fields like before?

Reply:
I added this wild card to exercise the unit test for the wild card scenario. The read operation should work when just with anonymous role without specifying any field. The example to exercise this case is as below. 
https://github.com/Azure/data-api-builder/blob/7efeec877b0ac53c3d40416096d03af43217fc05/config-generators/cosmosdb_nosql-commands.txt#LL5C9-L5C9
https://github.com/Azure/data-api-builder/blob/86b5917b6d04d8f78d537047cfefe97db1903042/src/Service.Tests/CosmosTests/QueryTests.cs#L248-L250